### ISSUE TITLE
Bug character stuck 

### ISSUE THREAD

**Issue by BarzoStrike** [suriyun-production/mmorpg-kit-docs#1730]
https://www.youtube.com/watch?v=QDLUywddwKU

Triggered when we use a skill that is standing still with press "w" (move) at the same time. Seems to be okay with the kind of skill we can walk. broken happens in both mmo and single. This video was shot in mmo

**Comment by insthync** [comment]
Only animation freezing?

**Comment by BarzoStrike** [comment]
watch the video it's not just visual. locks your position you can't talk to npc or pass through portal

**Comment by insthync** [comment]
As I can reproduce, it is only animation freezing issues.

**Comment by insthync** [comment]
Try to fix it by changing codes at line `670` to
```
stateUpdateData.clipSpeed = stateInfos[playingStateId].GetSpeed(moveAnimationSpeedMultiplier > 0f ? moveAnimationSpeedMultiplier : 1f);
```

**Comment by BarzoStrike** [comment]
Animation issue fixed. The real problem, character is stuck in a position. There are ongoing problems as in the video, the problem is only in mmo, there is no problem in single.

**Comment by insthync** [comment]
Fixed in 1.78, if it is still occurring, then reopen this issue post.

**Comment by BarzoStrike** [comment]
I tried everything seems fine. You are super
